### ISSUE TITLE
A11Y: Category Icons should be tabbable

### ISSUE THREAD

**Issue by comp615** [missive/emoji-mart#219]
The icons on top of the picker should be reachable in the tab order. There may also be an appropriate aria role for a nav menu or something similar that could be used
